README.md

```diff
@@ -11,21 +30,57 @@ google spreadsheet add-on.
 
 The project is based on [clasp](https://github.com/google/clasp) and [gas-local](https://github.com/mzagorny/gas-local) to enable local development and testing. Most of the Google functions has been mocked in order to work locally, check `gas_mock` for reference. The `src` directory contains the source code that will be pushed to the google apps webeditor. Changing code directly in the webeditor is not recommended apart from debugging because it will get wiped out next time someone pushes.
 
-### Prerequisites
+## Prerequisites
 
 * node.js >= 6.3.0
 * npm
 * npx : if your npm installation is old and you don't have npx, install it with: `sudo npm i -g npx`
 
-### Installation
+## Installation
 
 Install dependencies:
 
 ```bash
 $ npm install
 ```
 
-### Testing
+## Structure
+
+The structure of the project consists of the following files:
+
+### Queries
+
+They can be found in the ``src/api/queries`` folder. All the queries, sent to the API can be seen here. A quick note here, the getMetric query is more dynamic.
```

Comment:
> the getMetric query is more dynamic. 

what does more dynamic mean here?

Reply:
I'll remove that, don't really know what I was going for with this one 🤷‍♂️